lib/index.js

```diff
@@ -8,6 +8,14 @@ const miniget = require('miniget');
 const m3u8stream = require('m3u8stream');
 const parseTime = require('m3u8stream/dist/parse-time');
 
+const current = 'v4.0.5';
```

Comment:
this can be imported from `package.json`
```suggestion
const { version } = require('../package.json');
```